docs/installation/ca/Installing_CA.md

```diff
@@ -15,7 +15,8 @@ Additionally, make sure the FQDN has been [configured](../server/FQDN_Configurat
 CA Subsystem Installation
 -------------------------
 
-Prepare a file (e.g. ca.cfg) that contains the deployment configuration, for example:
+Prepare a file (e.g. ca.cfg) that contains the deployment configuration.
+A sample deployment configuration is available at [/usr/share/pki/server/examples/installation/ca.cfg](../../../base/server/examples/installation/ca.cfg).
```

Comment:
Question: This link seems redundant to the example that follows. At the first glance, they both carry different values. So, can we remove one of the entries to avoid confusing users? My $0.02 😄

Reply:
Fixed. I removed the sample configuration from the docs.